README.md

```diff
@@ -34,6 +34,25 @@ jobs:
 +        args: --config=.project.php_cs --diff --dry-run
 ```
 
+_to support fixing styles of a pull request via `/php-cs-fixer` keyword use:_
+
+```yml
+on:
+  issue_comment:
```

Comment:
Why on issue comment and not on push/pull_request?

Reply:
Ah you want to support “/please fix me” right?

Reply:
Exactly